packages/neon/neon/lib/src/pages/app_implementation_settings.dart

```diff
@@ -0,0 +1,183 @@
+    final syncBloc = Provider.of<SyncBloc>(context, listen: false);
+    final syncImplementation = appImplementation.getSync();
+
+    final appBar = AppBar(
+      title: Text(appImplementation.name(context)),
+      actions: [
+        IconButton(
+          onPressed: () async {
+            if (await showConfirmationDialog(
+              context,
+              AppLocalizations.of(context).settingsResetForConfirmation(appImplementation.name(context)),
+            )) {
+              appImplementation.options.reset();
+            }
+          },
+          tooltip: AppLocalizations.of(context).settingsResetFor(appImplementation.name(context)),
+          icon: Icon(MdiIcons.cogRefresh),
+        ),
+      ],
+    );
+
+    final body = SettingsList(
+      categories: [
+        for (final category in [...appImplementation.options.categories, null]) ...[
+          if (appImplementation.options.options.where((final option) => option.category == category).isNotEmpty) ...[
+            SettingsCategory(
+              title: Text(
+                category != null ? category.name(context) : AppLocalizations.of(context).optionsCategoryOther,
+              ),
+              tiles: [
+                for (final option
+                    in appImplementation.options.options.where((final option) => option.category == category)) ...[
+                  if (option is ToggleOption) ...[
+                    CheckBoxSettingsTile(
+                      option: option,
+                    ),
+                  ] else if (option is SelectOption) ...[
+                    DropdownButtonSettingsTile(
+                      option: option,
+                    ),
+                  ],
+                ],
+              ],
+            ),
+          ],
+        ],
+        if (syncImplementation != null) ...[
+          StreamBuilder(
+            stream: syncBloc.mappingStatuses,
+            builder: (final context, final mappingStatuses) => !mappingStatuses.hasData
+                ? const SizedBox.shrink()
+                : SettingsCategory(
+                    title: Text(AppLocalizations.of(context).optionsCategorySync),
+                    tiles: [
+                      for (final mappingStatus in mappingStatuses.requireData.entries
+                          .where((final mappingStatus) => mappingStatus.key.appId == appImplementation.id)) ...[
+                        CustomSettingsTile(
+                          title: Text(mappingStatus.key.title),
+                          subtitle: Text(mappingStatus.key.subtitle),
+                          leading: NeonUserAvatar(
+                            account: accountsBloc.accounts.value
+                                .singleWhere((final account) => account.id == mappingStatus.key.accountId),
+                            showStatus: false,
+                          ),
+                          trailing: IconButton(
```

Comment:
IconButtons should have a tooltip for a11y

Reply:
Also added labels to the icon itself.